MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -184,7 +174,7 @@ object VisionManager : ARDataProvider {
      * You should [destroy] when Vision SDK is no longer needed to release all resources.
      * No-op if called while SDK is created already.
      */
-    fun create() {
+    fun create(videoSource: VideoSource = CameraVideoSourceImpl(application)) {
```

Comment:
External video source is passed here. Default is still good old camera2 implementation.